docs/uax-with-aj1/Request-to-revise-UAX-50-for-the-harmonization-with-Adobe-Japan1.md

```diff
@@ -8,9 +15,7 @@
 * Shinyu MURAKAMI (Vivliostyle)
 * MURATA Makoto (Keio University)
 
-***
-
-[CITPC(Council for the Promotion of Character Information Technology)](https://moji.or.jp/) is a general incorporated association in Japan.  CITPC aims to enhance the interoperability of character information processing in Japan.  Most of the font vendors in Japan are members of CITPC.  
+## Background
 
 [UAX 50](https://www.unicode.org/reports/tr50/) was designed as a cornerstone of vertical writing for the Web and EPUB.  CITPC is grateful to the Unicode Consortium for the timely development of UAX 50 and its ongoing maintenance.   
 

```

Comment:
In the following para: "However, there are some inconsistencies...", replace "impacts of these ..." with "the impact of these..."